templates/vue/src/components/Lightbox/media/H5PMedia/H5PIframe.vue

```diff
@@ -349,11 +352,23 @@ export default {
                     h5pIframeComponent.node.id
                   )
                 }, 1000)
+              } else {
+                // Disable Autoplay For Youtube Workaround:
+                // There's a bug with the Youtube Video API such that you cannot disable autoplay.
+                // This is a workaround to stop the video upon loading.
+                if (!this.autoplay) {
+                  // As of April 2021, H5P has not implemented a stop functionality,
+                  // so there is no way to avoid the "More Videos" when paused.
+                  // This pause requires a timeout to avoid a spinner.
+                  setTimeout(() => {
+                    h5pVideo.pause()
+                  }, 1000)
```

Comment:
Instead of this, could we do a `setInterval` and check to see if the video is loaded and if it is loaded, then pause the video and clear the interval? If it's not loaded (i.e. cannot be paused), we should keep going until the video is paused. The reason I am suggesting is that for me the video loads faster than 1000ms and therefore I actually hear the video playing for a bit under a second.

Reply:
I tried using `setInterval` but it seems that by the time the H5P tells us the video has loaded as per loaded event found here: https://github.com/h5p/h5p-video, it may have already started playing. I opted to remove the `setTimeout` but there is a spinner.